renormalizer/mps/mps.py

```diff
@@ -696,11 +696,7 @@ def _evolve_dmrg_tdvp_vmf(self, mpo, evolve_dt) -> "Mps":
         else:
             coef = 1j
 
-        if not self.check_left_canonical():
-            self.move_qnidx(0)
-            self.left = True
-            self.canonicalise()
-            assert self.check_left_canonical()
+        self.ensure_left_canon()
```

Comment:
Is there any API for a user to change the default atol in checking the canonicalization when using VMF?

Reply:
Not yet but I've modified some code in this commit to make this easier. Regarding the check-canonical issue, I think the best option is to figure out why using atol=1e-3 in the first place and restore it to the default value.

Reply:
Actually I don't know. You told me that some tests will fail if atol is set to the default value.

Reply:
Indeed... I'll pay attention to these tests.